stretch_update.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
The first thing `stretch_update.sh` should do is reclone the stretch_install repository to ensure it's running the latest version of the script.

Reply:
Just added a git pull. That should be sufficient, right? Or a full clone is needed?

Reply:
Actually, a git pull should work just fine.